### ISSUE TITLE
Trust RockNSM GPG keys in yum

### ISSUE THREAD

**Issue by seven62** [rocknsm/rock#278]
```
TASK [sensor-common : Trust RockNSM GPG keys in yum] *********************************************************************************************************************************************
skipping: [simplerockbuild.simplerock.lan] => (item={u'test': False, u'repoid': u'rocknsm_2_1'}) 
skipping: [simplerockbuild.simplerock.lan] => (item={u'test': False, u'repoid': u'rocknsm_2_1-source'}) 
skipping: [simplerockbuild.simplerock.lan] => (item={u'test': False, u'repoid': u'rocknsm-testing'}) 
 [WARNING]: Consider using the yum module rather than running yum.  If you need to use command because yum is insufficient you can add warn=False to this command task or set
command_warnings=False in ansible.cfg to get rid of this message.

failed: [simplerockbuild.simplerock.lan] (item={u'test': True, u'repoid': u'rocknsm-local'}) => {"changed": false, "cmd": ["yum", "-q", "makecache", "-y", "--disablerepo=*", "--enablerepo=rocknsm-local"], "delta": "0:00:00.140865", "end": "2018-08-13 18:58:13.849699", "item": {"repoid": "rocknsm-local", "test": true}, "msg": "non-zero return code", "rc": 1, "start": "2018-08-13 18:58:13.708834", "stderr": "file:///srv/rocknsm/repodata/repomd.xml.asc: [Errno 14] curl#37 - \"Couldn't open file /srv/rocknsm/repodata/repomd.xml.asc\"\nTrying other mirror.\n\n\n One of the configured repositories failed (ROCKNSM Local Repository),\n and yum doesn't have enough cached data to continue. At this point the only\n safe thing yum can do is fail. There are a few ways to work \"fix\" this:\n\n     1. Contact the upstream for the repository and get them to fix the problem.\n\n     2. Reconfigure the baseurl/etc. for the repository, to point to a working\n        upstream. This is most often useful if you are using a newer\n        distribution release than is supported by the repository (and the\n        packages for the previous distribution release still work).\n\n     3. Run the command with the repository temporarily disabled\n            yum --disablerepo=rocknsm-local ...\n\n     4. Disable the repository permanently, so yum won't use it by default. Yum\n        will then just ignore the repository until you permanently enable it\n        again or use --enablerepo for temporary usage:\n\n            yum-config-manager --disable rocknsm-local\n        or\n            subscription-manager repos --disable=rocknsm-local\n\n     5. Configure the failing repository to be skipped, if it is unavailable.\n        Note that yum will try to contact the repo. when it runs most commands,\n        so will have to try and fail each time (and thus. yum will be be much\n        slower). If it is a very temporary problem though, this is often a nice\n        compromise:\n\n            yum-config-manager --save --setopt=rocknsm-local.skip_if_unavailable=true\n\nfailure: repodata/repomd.xml.asc from rocknsm-local: [Errno 256] No more mirrors to try.\nfile:///srv/rocknsm/repodata/repomd.xml.asc: [Errno 14] curl#37 - \"Couldn't open file /srv/rocknsm/repodata/repomd.xml.asc\"", "stderr_lines": ["file:///srv/rocknsm/repodata/repomd.xml.asc: [Errno 14] curl#37 - \"Couldn't open file /srv/rocknsm/repodata/repomd.xml.asc\"", "Trying other mirror.", "", "", " One of the configured repositories failed (ROCKNSM Local Repository),", " and yum doesn't have enough cached data to continue. At this point the only", " safe thing yum can do is fail. There are a few ways to work \"fix\" this:", "", "     1. Contact the upstream for the repository and get them to fix the problem.", "", "     2. Reconfigure the baseurl/etc. for the repository, to point to a working", "        upstream. This is most often useful if you are using a newer", "        distribution release than is supported by the repository (and the", "        packages for the previous distribution release still work).", "", "     3. Run the command with the repository temporarily disabled", "            yum --disablerepo=rocknsm-local ...", "", "     4. Disable the repository permanently, so yum won't use it by default. Yum", "        will then just ignore the repository until you permanently enable it", "        again or use --enablerepo for temporary usage:", "", "            yum-config-manager --disable rocknsm-local", "        or", "            subscription-manager repos --disable=rocknsm-local", "", "     5. Configure the failing repository to be skipped, if it is unavailable.", "        Note that yum will try to contact the repo. when it runs most commands,", "        so will have to try and fail each time (and thus. yum will be be much", "        slower). If it is a very temporary problem though, this is often a nice", "        compromise:", "", "            yum-config-manager --save --setopt=rocknsm-local.skip_if_unavailable=true", "", "failure: repodata/repomd.xml.asc from rocknsm-local: [Errno 256] No more mirrors to try.", "file:///srv/rocknsm/repodata/repomd.xml.asc: [Errno 14] curl#37 - \"Couldn't open file /srv/rocknsm/repodata/repomd.xml.asc\""], "stdout": "", "stdout_lines": []}
	to retry, use: --limit @/opt/rocknsm/rock/playbooks/site.retry
```

**Comment by seven62** [comment]
This issue is _**temporarily**_ resolved by setting both gpgcheck entries in `/opt/rocknsm/rock/playbooks/roles/sensor-common/tasks/config.yml` to `0`:  

```
179 - name: Setup local offline repo
180   yum_repository:
181     name: rocknsm-local
182     description: ROCKNSM Local Repository
183     baseurl: "{{ rocknsm_local_baseurl }}"
184     gpgcheck: 0
185     gpgkey: file:///etc/pki/rpm-gpg/RPM-GPG-KEY-RockNSM-2
186     repo_gpgcheck: 0
187     cost: 500
188   when: "{{ not rock_online_install | bool }}"
```

**Comment by jeffgeiger** [comment]
Fixed in the fix for #279
